src/main/java/network/brightspots/rcv/OutputWriter.java

```diff
@@ -981,6 +984,7 @@ private Map<String, Object> generateCvrSnapshotMap(
             entry("@type", "CVR.CVRContest"));
 
     return Map.ofEntries(
+        // TODO: Do we want this to use the getSuppliedId or the getId method?
         entry("@id", generateCvrSnapshotId(sanitizeStringForOutput(cvr.getId()), round)),
```

Comment:
Our choice here should be the same as in the `TODO` comment added before the `sanitizedId` definition.